src/balancerd/src/codec.rs

```diff
@@ -38,6 +39,7 @@ impl From<ErrorResponse> for BackendMessage {
 
 /// A connection that manages the encoding and decoding of pgwire frames.
 pub struct FramedConn<A> {
+    pub uuid: Uuid,
```

Comment:
What's the benefit of storing this here, vs in the function that handles the connection? It doesn't seemed to be used on `Drop` or anything.

Reply:
I don't really have a good reason. I've moved this to the function that handles the connection like you suggested.